src/fsharp/CompileOps.fs

```diff
@@ -1046,8 +1046,7 @@ let OutputPhasedErrorR (os:System.Text.StringBuilder) (err:PhasedError) =
                                          | NONTERM_Category_Definition -> Parser.NONTERM_moduleDefn
                                          | NONTERM_Category_Interaction -> Parser.NONTERM_interaction
                                          | nt -> nt)
-                            |> Set.ofList 
-                            |> Set.toList with 
+                            |> List.distinct with 
```

Comment:
Ah, ``Set.ofList + Set.toList`` also sorts the list, so this will have changed error messages